dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -96,7 +96,7 @@ def new_post
       enrollment = Enrollment.find_by!(code: params[:enrollment_code])
       workshop = enrollment.workshop
 
-      return redirect_to :thanks if WorkshopDailySurvey.exists?(user: current_user, pd_workshop: workshop, day: 5)
+      return redirect_to :pd_workshop_survey_thanks if WorkshopDailySurvey.exists?(user: current_user, pd_workshop: workshop, day: 5)
```

Comment:
Thanks for jumping in and fixing this @Hamms! Actually this is supposed to be the `thanks` action on this controller. It should be `redirect_to action: :thanks` like [here](https://github.com/code-dot-org/code-dot-org/blob/d9155e6156ab2d205b27c0fd05e8f522e2f14336/dashboard/app/controllers/pd/workshop_daily_survey_controller.rb#L70) but for now this `pd_workshop_survey_thanks` will be just fine.

I'll make a note to updated it when I return.

Reply:
And, of course that is just another way of referencing the same route so never mind. This is exactly right :)

Reply:
ah, yeah `action: :thanks` is probably more semantically correct, but I was rushing